src/Rings/groebner.jl

```diff
@@ -137,7 +141,7 @@ negdegrevlex([x, y])
 ```
 """
 function standard_basis(I::MPolyIdeal; ordering::MonomialOrdering = default_ordering(base_ring(I)),
-	complete_reduction::Bool = false, algorithm::Symbol = :buchberger)
+	complete_reduction::Bool = false, algorithm::Symbol = :buchberger, weights::Vector{E} = ones(Int32, ngens(base_ring(I)))) where {E <: Integer}
```

Comment:
We do not add `weights` to `standard_basis()`, same holds for `groebner_basis()`. We need to compute this data internally.